### DISCUSSION TITLE
How to ensure that existing data in S3 bucket syncs to a new milvus deployment

### DISCUSSION THREAD

**Discussion by abmygate** [milvus-io/milvus#25217] [Q&A and General discussion]
I had a milvus standalone deployment with some data. I have setup S3 as the storage driver and there were no errors while writing and searching. After a while, I had to Milvus moved to a different instance (Milvus runs in a standalone mode on an ECS cluster). I'm not seeing my old collection and data on the new deployment. Is there some additional setup that needs to be done to sync the data that's already present in the S3? Please help me sync the data, thanks in advance

**Comment by yhmo** [comment]
We recommend using this tool to backup/restore data from a Milvus instance to another: https://github.com/zilliztech/milvus-backup

**Reply by abmygate** [reply]
milvus-backup requires milvus-cluster, but my setup is standalone. Can milvus not do the sync as soon as a new instance comes up and connects to an existing s3 collection?

**Reply by yhmo** [reply]
milvus-backup also works for standalone milvus.
